### ISSUE TITLE
YouTube embed iframe is missing title attribute

### ISSUE THREAD

**Issue by BlueSlug** [inclusive-design/wecount.inclusivedesign.ca#672]
## Describe the bug

The YouTube embedded video `iframe` element is missing the `title` attribute, which affects accessibility.

## To Reproduce

1. Go to https://wecount.inclusivedesign.ca/
2. Inspect the markup for the embedded YouTube video
3. See the error

## Expected behavior

The `iframe` element should have a `title` attribute set with meaningful information.

## Screenshots

n/a

## Desktop (please complete the following information):

n/a

## Smartphone (please complete the following information):

n/a

## Additional context

W3C information explaining accessibility concerns:
https://www.w3.org/TR/WCAG20-TECHS/H64.html

Affected WCAG Success Criteria:
[2.4.1 Bypass Blocks](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/navigation-mechanisms-skip.html)
[4.1.2 Name, Role, Value](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/ensure-compat-rsv.html)
